authorization-common/src/main/java/com/sequenceiq/authorization/utils/EventAuthorizationDto.java

```diff
@@ -0,0 +1,33 @@
+package com.sequenceiq.authorization.utils;
+
+public class EventAuthorizationDto {
+
+    private String resourceCrn;
+
+    private String resourceType;
+
+    public EventAuthorizationDto(String resourceCrn, String resourceType) {
+        this.resourceCrn = resourceCrn;
+        this.resourceType = resourceType;
```

Comment:
Is the `resourceType` expected to not be null? We will get a strange NPE in `EventAuthorizationUtils` if not.